tests/console_test.go

```diff
@@ -111,9 +104,8 @@ var _ = Describe("[rfe_id:127][posneg:negative][crit:medium][vendor:cnv-qe@redha
 				}
 
 				table.DescribeTable("should return that we are running alpine", func(createVMI vmiBuilder) {
-					vmi, dv := createVMI()
```

Comment:
Can we actually modify the `vmiBuilder` please?

Reply:
Done.